tests/yamlload.py

```diff
@@ -30,7 +30,7 @@ def update_debug_window(
 def main() -> None:
     """Main method"""
 
-    with open("tests/environment.yaml", "r") as file:
```

Comment:
These are meant to be run from project root, so `tests/` is the correct path.

Reply:
I doesn't work when U tried

Reply:
These files aren't really tests anyways, just files I work on to see if things are alright. If you run the file from the same directory as `setup.py` it works, and that is the way it is meant to be ran.

Reply:
Alright. I changed it back